src/outputs/outputs.cpp

```diff
@@ -1069,3 +1069,15 @@ void OutputType::CalculateCartesianVector(AthenaArray<Real> &src, AthenaArray<Re
   }
   return;
 }
+
+bool OutputType::ContainVariable(const std::string &haystack, const std::string &needle) {
+  if (haystack.compare(needle) == 0)
+    return true;
+  if (haystack.find(',' + needle + ',') != std::string::npos)
+    return true;
+  if (haystack.find(needle + ',') == 0)
+    return true;
+  if (haystack.find(',' + needle) == haystack.length() - needle.length())
```

Comment:
This ensure that `...,b` is not considered a match for `...,b3`, right?

Reply:
Yes, that's the intention... although just found that the index was off by one.
Just fixed it in the last commit.